src/prefabs/pageWithCrudDialogs.tsx

```diff
@@ -211,6 +212,34 @@ const beforeCreate = ({
     return boxPrefab;
   };
 
+  const disabledKinds = createBlacklist([
```

Comment:
login token should probably not have to be whitelisted here, object property likely not supported yet as well